src/api/todos.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodos = (data: Omit<Todo, 'id'>) => {
```

Comment:
It's better to use the correct function name for adding a todo. Please change 'postTodos' to 'addTodo'.